package.json

```diff
@@ -266,6 +266,6 @@
     "src",
     "LICENSE",
     "README.md",
-    "types.ts"
```

Comment:
Just to double-check but did you mean to remove `types.ts`? Description read adding changelog.

Reply:
The file does not exist.